indico/modules/users/client/js/dashboard.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import ICSCalendarLink from 'indico/modules/users/react/components/ICSCalendarLink';
+
+import {Translate} from 'indico/react/i18n';
+
+document.addEventListener('DOMContentLoaded', () => {
+  const userId = document.querySelector('body').dataset.userId;
```

Comment:
Maybe it is better to use `Indico.User.id` instead of searching the `DOM`

Reply:
hm when all you need is the ID, I think using the dataset is fine (but: `document.body.dataset.userId`); feels cleaner to me than using that global